### PR TITLE
Docs: Fix indentation of snippets

### PR DESCRIPTION
**Pull request by jrudolph**

### CODE REVIEW

File: docs/src/main/paradox/_template/page.st [hunk 1]
```diff
@@ -71,10 +71,10 @@
 
             <div class="page-content row">
               <div class="small-12 large-9 column" id="docs">
-                $page.content$
-                
+$page.content$
```
Comment (ktoso) [inline docs/src/main/paradox/_template/page.st]: add a comment that this is on purpose?

Comment (jrudolph) [inline docs/src/main/paradox/_template/page.st]: Good point, added a comment.

Comment (jrudolph) [inline docs/src/main/paradox/_template/page.st]: FYI: `$! comment !$` can be used to add comments on the ST4 level which won't be added to the output.

File: docs/src/main/paradox/_template/page.st [hunk 2]
```diff
@@ -71,10 +71,15 @@
...
             <div class="page-content row">
               <div class="small-12 large-9 column" id="docs">
-                $page.content$
-                
+
+$!
+   Must not be indented, will otherwise ST4 will otherwise indent <pre> blocks in the
+   content as well, leading to extra indentation in the generated output.
+!$
+$page.content$
```
Comment (ktoso) [inline docs/src/main/paradox/_template/page.st]: cool, thanks!

**Pull request merged**